packages/block-library/src/comment-content/block.json

```diff
@@ -35,12 +35,25 @@
 				"fontSize": true
 			}
 		},
+		"__experimentalBorder": {
+			"radius": true,
+			"color": true,
+			"width": true,
+			"style": true,
+			"__experimentalDefaultControls": {
+				"radius": true,
+				"color": true,
+				"width": true,
+				"style": true
+			}
+		},
 		"spacing": {
 			"padding": [ "horizontal", "vertical" ],
 			"__experimentalDefaultControls": {
 				"padding": true
 			}
 		},
-		"html": false
+		"html": false,
+		"style": "wp-block-comment-content"
```

Comment:
There is no field called `supports.style` - we'll need to add it at the root level of the json object, just like any other block.

Reply:
Thank you for the feedback. I will update changes.

Reply:
Added style at root level